enos/modules/docker_openssh_server/main.tf

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+terraform {
+  required_providers {
+    docker = {
+      source  = "kreuzwerker/docker"
+      version = "3.0.1"
+    }
+
+    tls = {
+      source  = "hashicorp/tls"
+      version = "4.0.4"
+    }
+  }
+}
+
+variable "network_name" {
+  description = "Name of Docker Network"
+  type        = string
+}
+variable "container_name" {
+  description = "Name of Docker Container"
+  type        = string
+  default     = "openssh-server"
+}
+variable "target_user" {
+  description = "SSH username for target"
+  type        = string
+  default     = "ubuntu"
+}
+variable "private_key_file_path" {
+  description = "Local Path to key used to SSH onto created hosts"
+  type        = string
+}
+
+locals {
+  is_openssh = length(regexall(".*OPENSSH.*", file(var.private_key_file_path))) > 0
```

Comment:
What is this used for?

Reply:
😅 Yup. No longer used. Originally had some logic to parse the key differently if it didn't say OPENSSH (i.e. RSA), but it wasn't necessary (mostly due to my lack of understanding of key encryption terminology).